test/basedownloadcontroller-tests.js

```diff
@@ -0,0 +1,379 @@
+/**
+ * Copyright 2023 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require('chai').assert;
+const Controller = require('../lib/MockDownloadController');
+const clone = require('clone');
+const objectPath = require('object-path');
+
+let kubeData = {};
```

Comment:
```suggestion
const kubeData = {};
```

Reply:
gets reset in afterEach, so can't change to const